### DISCUSSION TITLE
How to get the PRO tag on Your Account

### DISCUSSION THREAD

**Discussion by Alien-Alfa** [community/community#20659] [Other Feature Feedback, Questions, & Ideas]
I have seen many people with the Tag PRO on their accounts I wana know how do they get it

<img width="238" alt="Screenshot 2022-07-16 at 3 01 30 PM" src="https://user-images.githubusercontent.com/64305844/179349273-440a7b13-34d3-406e-a641-143bf9a2948a.png">

**Comment by palamarchukser** [comment]
duplicated, check this https://github.com/github-community/community/discussions/19414

**Reply by Alien-Alfa** [reply]
Oh Sorry, Thanks Btw
